apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/HelpBox.tsx

```diff
@@ -1,62 +1,62 @@
 import { Paper, withStyles } from '@material-ui/core';
 import { ClassNameMap } from '@material-ui/core/styles/withStyles';
+import { ImageList, ImageListItem, Typography } from '@mui/material';
 
 const styles = {
     container: {
         padding: 12,
         marginBottom: '10px',
         marginRight: '5px',
     },
```

Comment:
Use MUIv5's [`sx`](https://mui.com/system/getting-started/the-sx-prop/) prop instead of `withStyles`